nessie/models/student_schema.py

```diff
@@ -43,22 +43,18 @@ def staging_schema():
 
 
 def drop_staged_enrollment_term(term_id):
-    with redshift.transaction() as transaction:
-        delete_sql = 'DELETE FROM {schema}.{table} WHERE term_id = %s'
-        transaction.execute(
-            delete_sql,
-            schema=psycopg2.sql.Identifier(staging_schema()),
-            table=psycopg2.sql.Identifier('student_enrollment_terms'),
-            params=(term_id,),
-        )
+    redshift.execute(
+        'DELETE FROM {schema}.{table} WHERE term_id = %s',
+        schema=psycopg2.sql.Identifier(staging_schema()),
+        table=psycopg2.sql.Identifier('student_enrollment_terms'),
+        params=(term_id,),
```

Comment:
@paulkerschen Is trailing comma, after `term_id`, necessary?

Reply:
@johncrossman Actually yes; it's a Python quirk with one-element tuples. https://ubuntuforums.org/archive/index.php/t-2232993.html